File: 프로그래머스/lv0/120882. 등수 매기기/등수 매기기.java
import java.util.*;
class Solution {
    public int[] solution(int[][] score) {
        int[] answer = new int[score.length];
        List<Integer> avg2 = new ArrayList<>(); 
        
        for (int i = 0; i < score.length; i++) {
            avg2.add(score[i][0] + score[i][1]);
        }
        avg2.sort(Collections.reverseOrder());
        for(int i = 0; i < score.length; i++) {
            answer[i] = avg2.indexOf(score[i][0] + score[i][1]) + 1;
        }
        return answer;
    }
}

File: 프로그래머스/lv0/120882. 등수 매기기/등수 매기기.js
function solution(score) {
    var answer = [];
    var avg = [];
    for(let i =0; i< score.length; i++) {
        const sum = score[i].reduce((a,b) => a+b);
        avg.push(sum/2); 
    }

    let tmp = avg.slice().sort(function(a,b){return b-a});
    console.log(tmp);
    for(let i =0; i < avg.length; i++) {
        answer.push(tmp.indexOf(avg[i])+1);
    }
    
    return answer;
}